server/app/main.py

```diff
@@ -19,3 +27,34 @@ async def mlmd_push(info: Request):
         'data':req_info
     }
 
+@app.get("/mlmd_pull",response_class=HTMLResponse)
+async def mlmd_pull(request: Request):
```

Comment:
Remove all the hardcoding in this file and create a file called server config and read from that